ui/user/src/lib/components/navbar/KnowledgeUpload.svelte

```diff
@@ -72,6 +72,6 @@
 			<Upload class="size-4" />
 		{/if}
 		Upload (.pdf, .pptx, .txt)
```

Comment:
```suggestion
		Upload
```

Reply:
Changed, all added the rest of the types that knowledge supports.